cupy/cuda/cupy_rtc.h

```diff
@@ -0,0 +1,15 @@
+#ifndef INCLUDE_GUARD_CUPY_RTC_H
+#define INCLUDE_GUARD_CUPY_RTC_H
+
+#if CUPY_USE_HIP
+
+#include "cupy_hiprtc.h"
+
+#else
+
+#include "cupy_nvrtc.h"
+
+#endif
```

Comment:
I guess the empty lines were reserved for inserting more functions in the future? Would be nice to remove them, though.